easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'Bundle'
+
+name = 'IPython'
+version = '3.2.3'
+
+homepage = 'http://ipython.org/index.html'
+description = """IPython provides a rich architecture for interactive computing with:
+ Powerful interactive shells (terminal and Qt-based).
+ A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
+ Support for interactive data visualization and use of GUI toolkits.
+ Flexible, embeddable interpreters to load into your own projects.
+ Easy to use, high performance tools for parallel computing."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+python = 'Python'
+pyver = '2.7.10'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('PyZMQ', '15.1.0', '%s-zmq4' % versionsuffix),
+]
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+exts_list = [
+    ('pysqlite', '2.8.1', {
+        'modulename': 'pysqlite2',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pysqlite/'],
+    }),
+    ('requests', '2.9.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests/'],
+    }),
+    ('Pygments', '2.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/P/Pygments/'],
+    }),
+    ('tornado', '4.3', {
```

Comment:
this requires `backports-abc`, please include it as an extra extension above this one